### PR TITLE
Added guildroster Component

### PR DESCRIPTION
**Pull request by djstedman**
Added guildroster component and images to support it
lists raiders based on data pulled from /v1/guild/members
as returned from the server API

API will need to be updated or else this will fail

### CODE REVIEW

File: client/app/boot.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
 import {bootstrap}    from 'angular2/platform/browser';
 import {AppComponent} from './app.component';
-bootstrap(AppComponent);
+import {HTTP_PROVIDERS}    from 'angular2/http';
```
Comment (jsmylnycky) [inline client/app/boot.ts]: This stuff actually should be in client/app/app.component.ts, as that is the all encompassing component of the site.

File: client/app/components/guildroster/guildroster.service.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import {Injectable} from 'angular2/core';
```
Comment (jsmylnycky) [inline client/app/components/guildroster/guildroster.service.ts]: This file should be in /client/app/services and conform in structure to the existing ones.

**Comment by jsmylnycky** [comment]
Just fix up the two things about where the new files are included, and the placement of the service (and obviously any reference paths to that file), and we should be good :)

Let me know when you update and i'll get a final review in before we do the merge.

**Comment by djstedman** [comment]
I modified things to be where you said.. think it should be good now, however until the api is active on the server this probably shouldnt be merged anyhow

**Comment by jsmylnycky** [comment]
Ok...

I've got the API up and running with the guild members endpoint returning a list of all raiders:

https://api.nobreaksguild.com/v1/guild/members

Feel free to give this a test and then merge whenever you feel ready.

**Pull request merged**